src/components/Loader/Loader.jsx

```diff
@@ -10,42 +12,82 @@ const Loader = forwardRef(
       svgClassName,
       className,
       size,
+      color,
+      type,
       id
     },
     ref
   ) => {
     const overrideClassName = backwardCompatibilityForProperties([className, svgClassName], "");
 
-    const style = useMemo(() => {
+    const sizeStyle = useMemo(() => {
       if (typeof size === "number") {
         return { width: size, height: size };
       }
       return undefined;
     }, [size]);
 
     return (
-      <div className="monday-loader-component" ref={ref} role="alert" title="loading" style={style} id={id}>
-        <svg className={`circle-loader-spinner ${overrideClassName}`} viewBox="0 0 50 50">
-          <circle className="circle-loader-spinner-path" cx="25" cy="25" r="20" fill="none" strokeWidth="5" />
+      <div
+        className={cx(styles.mondayLoaderComponent, "monday-loader-component")}
```

Comment:
while naming css modules class name no need to add "monday" or "component" because of the way we config the css module make out class names unique.
please change the **css module class name** to loaderWrapper or something similar